assets/ar/factuality_disinformation_harmful_content/subjectivity/Subjectivity_GPT4_FewShot.py

```diff
@@ -0,0 +1,82 @@
+from llmebench.datasets import ThatiARDataset
+from llmebench.models import OpenAIModel
+from llmebench.tasks import SubjectivityTask
+
+
+def metadata():
+    return {
+        "author": "Arabic Language Technologies, QCRI, HBKU",
+        "model": "gpt-4-8k",
+        "description": "GPT4 8k tokens model hosted on Azure, using the ChatCompletion API. API version '2023-03-15-preview'. 3 samples where chosen per test sample based on MaxMarginalRelevance for few shot learning.",
+        "scores": {"Macro-F1": "0.745"},
+    }
+
+
+def config():
+    return {
+        "dataset": ThatiARDataset,
+        "task": SubjectivityTask,
+        "model": OpenAIModel,
+        "model_args": {
+            "class_labels": ["SUBJ", "OBJ"],
+            "max_tries": 30,
+        },
+        "general_args": {
+            "test_split": "ar/test",
+            "fewshot": {"train_split": "ar/train"},
```

Comment:
In ThatiARDataset, we don't have "ar" split so I think this might fail, and you don't need to specify the test/train splits as the fraemwork will automatically find them.